handlers/insolvency_resource.go

```diff
@@ -104,6 +104,80 @@ func HandleCreateInsolvencyResource(svc dao.Service) http.Handler {
 	})
 }
 
+// HandleSubmitAttachment receives an attachment to be stored against the Insolvency case
+func HandleSubmitAttachment(svc dao.Service) http.Handler {
```

Comment:
I'd move this handler to it's own file. I think you were planning on doing this as you have an empty `attachment_test.txt` file